app/views/session.py

```diff
@@ -50,6 +51,8 @@ def login():
         jti_claim_storage.use_jti_claim(jti_claim)
     except JtiTokenUsed as e:
         raise Unauthorized from e
+    except (TypeError, ValueError) as e:
+        raise InvalidTokenException
```

Comment:
This should be `raise InvalidTokenException from e` to keep the traceback from the original exception.